packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -45,8 +45,8 @@
                 grid-template-columns: 1.2fr 1fr;
                 &--turbos {
                     display: flex;
-                    flex-direction: row;
-                    border-bottom: none !important;
```

Comment:
I checked how the card looked like without this property and nothing changed after I'd deleted it.  I also checked nested styles and it turned out, that that element have no border, nested from parents. I need an explanation of why `!important` was used, because maybe it affects something else (which I haven't noticed).